gadget/position.go

```diff
@@ -92,6 +92,13 @@ type PositionedContent struct {
 	Index int
 }
 
+func (p PositionedContent) String() string {
+	if p.Image != "" {
+		return fmtIndexAndName(p.Index, fmt.Sprintf("%v@%#x{%v}", p.Image, p.StartOffset, p.Size))
+	}
+	return fmtIndexAndName(p.Index, fmt.Sprintf("source:%s", p.Source))
```

Comment:
out of idle curiosity I looked at `fmtIndexAndName`, and it puts the second argument in a `(%q)`, that is, between parentheses and quotes. Is that what we want?

Reply:
The names can contain spaces. I can tweak the code to quote just the name part.